OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -486,11 +480,14 @@ Action InitHotkeysPanel(Widget panel)
 					var added = new HashSet<HotkeyDefinition>();
 					var template = templates.Get(templateNode.Value.Value);
 					var remapDialogRoot = panel.Get("HOTKEY_DIALOG_ROOT");
-					var remapDialogPlaceholder = panel.GetOrNull("HOTKEY_DIALOG_PLACEHOLDER");
+
 					foreach (var t in types)
```

Comment:
Please add braces around these other blocks so the inner one isn't left hanging.